python/cudf_polars/cudf_polars/streaming/actor_graph/utils.py

```
@@ -166,7 +166,7 @@ async def shutdown_on_error(
     # Create tracer only if LOG_TRACES is enabled and IR is provided
     tracer: ActorTracer | None = None
     contextvars: dict[str, Any] = {}
-    from cudf_polars.experimental.rapidsmpf.tracing import ActorTracer
+    from cudf_polars.streaming.actor_graph.tracing import ActorTracer
```

Ruff (0.15.13)

[warning] `import` should be at the top-level of a file

(PLC0415)